### ISSUE TITLE
meta.title and meta.description only renders first character

### ISSUE THREAD

**Issue by martinbira** [squidfunk/mkdocs-material#553]
## Description
The content in `<title></title>` and `<description></description>` derived from the page meta  `title` and `description` only consist of the first character for the page meta value.

### Expected behavior

`<title></title>` should consist of full content from the page meta `title`
`<description></description>` should consist of full content from the page meta `description`

### Actual behavior

`<title></title>` consist of only the first character from the page meta `title`
`<description></description>` consist of only the first character from the page meta `description`

### Steps to reproduce the bug

1. In `page.md` set the following meta data:
```
---
title: Page title
description: Page description
---
```
2. Verify that mkdocs builds page.
3. Check page source for `<title>P</title>` and `<description>P</description>`

### Package versions

* Python: 3.6.1
* MkDocs: 0.17.1
* Material: 2.01

### System information

* OS: Windows 10

**Comment by coliff** [comment]
I could reproduce this issue but it's easily fixed by using the following:
```
---
page.title: Page title
page.description: Page description
---
```
I got those variable names from the MkDocs Release Notes:
http://www.mkdocs.org/about/release-notes/#version-0170-2017-10-19

**Comment by coliff** [comment]
An update from me. I realised that using the above does correctly populates the title and description - but it also adds the title and description within the HTML of the page. Looks like a bug.

Using the following works though:
```
---
page_title: Page title
page_description: Page description
---
```

**Comment by squidfunk** [comment]
Fixed in #554, release will follow today.

**Comment by StephenTaylor-Kx** [comment]
The same is true for author metadata. For example, 

    ---
    author: John Doe
    ---

yields 

    <meta name="author" content="J">

and the same workaround succeeds:

    ---
   page.author: John Doe
    ---

yields

    <meta name="author" content="John Doe">

**Comment by squidfunk** [comment]
Same fix as in #554 - I can't remember why I put the `first` modifier on it, maybe the format was different some time ago. Feel free to open a new issue for this so it doesn’t get forgotten.
